python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -108,22 +108,103 @@ def _apply_nms(orig_prediction: dict, iou_thresh: float = 0.5):
     return nms_prediction
 
 
-def _wrap_image_model(model, examples, model_task, is_function, number_of_classes=None):
+def _process_automl_detections_to_raw_detections(
+        image_detections,
+        label_dict: Dict[str, int],
+        image_size: Tuple[int, int]) -> Dict[str, Tensor]:
+    """Process AutoML mlflow object detections from a single image.
+
+    :param image_detections: AutoML mlflow detections
+    :type image_detections: list
+    :param label_dict: Label dictionary mapping class names to an index
+    :type label_dict: dict
+    :param image_size: Image size
+    :type image_size: tuple(int, int)
+    :return: Raw detections
+    :rtype: dict
+
+    The length of image_detections list will be equal to the number
```

Comment:
usually this description would go between the first line and the parameters, so on line 116 instead of 126

Reply:
fixed